### PR TITLE
fixed broken links to visualstudio/data-tools

### PR DESCRIPTION
**Pull request by mikeblome**
fixed broken links to visualstudio/data-tools topics. This is prefereable to creating redirects because the link text also needed to change in some cases. Le me know if i missed any.

also: These could be site-relative urls of the form: /visualstudio/data-tools/<topic> ?

@svick @mairaw

### CODE REVIEW

File: docs/visual-basic/developing-apps/accessing-data.md [hunk 1]
```diff
@@ -56,26 +56,26 @@ translation.priority.ht:
  [Work with datasets in n-tier applications](https://docs.microsoft.com/visualstudio/data-tools/work-with-datasets-in-n-tier-applications)  
  Provides links to topics about how to create multitiered data applications.  
      
- [Connecting to Data in Visual Studio](https://docs.microsoft.com/visualstudio/data-tools/connecting-to-data-in-visual-studio)  
+ [Add new connections](https://docs.microsoft.com/en-us/visualstudio/data-tools/add-new-connections)
```
Comment (mairaw) [inline docs/visual-basic/developing-apps/accessing-data.md]: please remove en-us from the new links

**Review by mairaw — DISMISSED** [review]
There's a reason why we use the full path instead of the relative link: it works on GitHub preview. The relative path only works on docs.

**Comment by GuardRex** [comment]
Addresses https://github.com/dotnet/docs/issues/1815

**Comment by GuardRex** [comment]
@mikeblome LGTM ... the links are golden here. Are you ready to merge?

**Review by guardrex — APPROVED** [review]

**Comment by GuardRex** [comment]
@mikeblome I'm gonna send this one on its way. If you need to update further, shoot in a follow-up PR. Thanks for fixing the links. :rocket:

**Pull request merged**